WordPress/src/main/java/org/wordpress/android/ui/posts/EditPostSettingsFragment.java

```diff
@@ -104,9 +103,7 @@
     private static final int SELECT_LIBRARY_MENU_POSITION = 100;
     private static final int CLEAR_FEATURED_IMAGE_MENU_POSITION = 101;
 
-    private PostModel mPost;
-    private SiteModel mSite;
-
+    private PostSettingsListener mListener;
```

Comment:
Patch looks good, but the name `mListener` seems off (we're not notifying this object when something happens in this fragment), `something-delegate` maybe? @aforcier might have a better idea.